osbs/utils.py

```diff
@@ -289,6 +289,10 @@ def get_time_from_rfc3339(rfc3339):
         return timegm(time_tuple)
 
 
+
+VALID_BUILD_CONFIG_NAME_CHARS = re.compile('[a-z0-9]')
```

Comment:
Is it a good idea to remove every `-`? The only thing we should make sure, that filtered branch and repo names are not beginning or ending in `-`.